README.md

```diff
@@ -1,177 +1,61 @@
-| [12K Research](https://12k.co/) | ![12k](docs/src/readme_images/12k.png)| Media |   Document similarity analysis on media articles. |
-| [Stillwater Supercomputing](http://www.stillwater-sc.com/) | ![stillwater](docs/src/readme_images/stillwater.png) | Hardware | Document comprehension and association with word2vec. |
-| [SiteGround](https://www.siteground.com/) |  ![siteground](docs/src/readme_images/siteground.png) | Web hosting | An ensemble search engine which uses different embeddings models and similarities, including word2vec, WMD, and LDA. |
-| [Capital One](https://www.capitalone.com/) | ![capitalone](docs/src/readme_images/capitalone.png) | Finance | Topic modeling for customer complaints exploration. |
-
--------
-
-Citing gensim
-------------
-
-When [citing gensim in academic papers and theses], please use this
-BibTeX entry:
-
-    @inproceedings{rehurek_lrec,
-          title = {{Software Framework for Topic Modelling with Large Corpora}},
-          author = {Radim {\v R}eh{\r u}{\v r}ek and Petr Sojka},
-          booktitle = {{Proceedings of the LREC 2010 Workshop on New
-               Challenges for NLP Frameworks}},
-          pages = {45--50},
-          year = 2010,
-          month = May,
-          day = 22,
-          publisher = {ELRA},
-          address = {Valletta, Malta},
-          note={\url{http://is.muni.cz/publication/884893/en}},
-          language={English}
-    }
-
-  [citing gensim in academic papers and theses]: https://scholar.google.com/citations?view_op=view_citation&hl=en&user=9vG_kV0AAAAJ&citation_for_view=9vG_kV0AAAAJ:NaGl4SEjCO4C
-
-  [Travis CI for automated testing]: https://travis-ci.org/RaRe-Technologies/gensim
-  [design goals]: http://radimrehurek.com/gensim/about.html
-  [RaRe Technologies]: http://rare-technologies.com/wp-content/uploads/2016/02/rare_image_only.png%20=10x20
-  [rare\_tech]: //rare-technologies.com
-  [Talentpair]: https://avatars3.githubusercontent.com/u/8418395?v=3&s=100
-  [citing gensim in academic papers and theses]: https://scholar.google.cz/citations?view_op=view_citation&hl=en&user=9vG_kV0AAAAJ&citation_for_view=9vG_kV0AAAAJ:u-x6o8ySG0sC
-
-  
-  
-  [documentation and Jupyter Notebook tutorials]: https://github.com/RaRe-Technologies/gensim/#documentation
-  [Vector Space Model]: http://en.wikipedia.org/wiki/Vector_space_model
-  [unsupervised document analysis]: http://en.wikipedia.org/wiki/Latent_semantic_indexing
-  [NumPy and Scipy]: http://www.scipy.org/Download
-  [ATLAS]: http://math-atlas.sourceforge.net/
-  [OpenBLAS]: http://xianyi.github.io/OpenBLAS/
-  [source tar.gz]: http://pypi.python.org/pypi/gensim
-  [documentation]: http://radimrehurek.com/gensim/install.html
+This is a forked gensim version, which edits the default doc2vec model to support pretrained word2vec during training doc2vec. It forked from gensim 3.8.
```

Comment:
Write your documentation so that it's useful from the point of view of the reader.

"This is a forked gensim version" is not relevant to the user. Furthermore, it becomes misleading the moment we actually merge this PR.